Makefile

```diff
@@ -134,6 +134,36 @@ else
 	go test -v -timeout 3600s $(TEST_ARGS) ./test/e2e/...
 endif
 
+kind-e2e-test: build kustomize docker-build create-kind-multicluster kind-load-image-multi e2e-test
+
+single-up: build kustomize docker-build create-kind-cluster kind-load-image cert-manager
+	$(KUSTOMIZE) build scripts/control_plane | kubectl apply -f -
+
+multi-up: build kustomize docker-build create-kind-multicluster kind-load-image-multi cert-manager-multi
```

Comment:
Same as the `single-up` target, I don't think we should be deleting clusters here.

Reply:
Can I ask why not @jsanda ?  If this is primarily a development friendly script isn't it likely you're running it to start up a new test of some sort?  Maybe I'm missing a use-case for this?